docs/whats-new.rst

```diff
@@ -4,6 +4,12 @@ What's New
 v0.1.4 (? ? 2015)
 --------------------
 
+Added topography.
+
+Added automated linear stability analysis.
+
+Added multi layered model subclass. 
+
```

Comment:
Maybe I'm confused about the sqg model...I thought you changed something there too. Maybe I was thinking of this: f48d4e8f2a8dee86a434c2a6612e66a27271d38b

The point is, I don't know! All I know is that your first list was incomplete.